### PR TITLE
Impl [Projects] Make the main cross project counters clickable

### PR DESCRIPTION
**Pull request by adi-gini**
Jira link:
https://iguazio.atlassian.net/browse/ML-8120

### CODE REVIEW

File: src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss [hunk 1]
```diff
@@ -21,18 +27,44 @@
...
       justify-content: center;
     }
 
+    &__link {
+      color: $darkPurple;
+      cursor: pointer;
+
+      &:hover {
+        text-decoration: underline;
+        color: $bombay;
```
Comment (Taras-Hlukhovetskyi) [inline src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss]: It looks like in the Figma we have another color #7F7989 for hovered counters. Could you please double check it

File: src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss [hunk 2]
```diff
@@ -21,18 +27,44 @@
...
+        text-underline-offset: 1px;
+        text-decoration-thickness: 2px
```
Comment (Taras-Hlukhovetskyi) [inline src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss]: missed semicolon. Also please use style-lint to format CSS file

**Review by Taras-Hlukhovetskyi — CHANGES_REQUESTED** [review]

File: src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss (hunk 1 above)
Comment (adi-gini) [reply src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss]: Done

File: src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss (hunk 2 above)
Comment (adi-gini) [reply src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss]: Done

**Review by Taras-Hlukhovetskyi — APPROVED** [review]

**Pull request closed without merging**